README.md

```diff
@@ -13,11 +19,11 @@ For writing and editing code, we recommend using the **GitHub Flow** method of w
 Following this workflow is most important when working collaboratively on code which is intended for publication and applies even to minor, low-risk changes. There may be instances when you are working alone, and on code which is very developmental, in which case you may not need to follow this workflow. There are also specific repositories which do not require this workflow such as this repository and the [resources repository](https://github.com/NHS-NSS-transforming-publications/resources) - this is because changes to these repositories do not affect code for publications. 
 
 #### Top tips for following the GitHub Flow:
```

Comment:
Very minor but should this now be the same format as the other section headers as it's in the table of contents?